html/global_attributes.json

```
@@ -353,7 +353,7 @@
               "opera": "mirror",
               "opera_android": "mirror",
               "safari": {
-                "version_added": "≤13.1"
+                "version_added": "8"
```

It looks like this was already implemented via https://github.com/WebKit/WebKit/commit/43eb8b77729ac1b5af02f92e3e4ace7cf4992cba in WebKit 533.9 before Safari 5 (WebKit [533.16](https://github.com/mdn/browser-compat-data/blob/1c1dab811525635e6247d72290374843373eb527/browsers/safari.json#L59-L64)).

```suggestion
                "version_added": "5"
```